lib/browser/rpc.js

```diff
@@ -366,14 +382,14 @@ function sendRequest(command, data, host = sessionHost) {
                     error = `Unknown callback id: ${callback}`
                 }
             } catch (e) {
-                error = e.message || ('' + e);
+                error = e.message || ("" + e);
```

Comment:
```suggestion
                error = e.message || (`${e}`);
```

Reply:
This was an automatic change from applying the eslint rules of the project root folder. I would rather not spent too much time on editing this any futher :)

Reply:
well there is the `commit suggestion` button :)

Reply:
Just leave as it is - `"" + e` is just so much last year :smile:

Reply:
seriously, why not click the `commit suggestion` button